packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -398,6 +398,7 @@ describe('App: Settings', () => {
         showSystemNotificationStub = o.sinon.stub(ctx.actions.electron, 'showSystemNotification')
         ctx.coreData.localSettings.preferences.notifyWhenRunStarts = false
         ctx.coreData.localSettings.preferences.notifyWhenRunStartsFailing = true
+        ctx.coreData.localSettings.preferences.desktopNotificationsEnabled = true
```

Comment:
Worth adding an e2e test here to assert disabled states?

Reply:
The component test should be okay I think - if the elements have the `disabled` attribute (which they do, we assert so) I don't see a scenario where this behavior regresses and the component test doesn't catch it.